### PR TITLE
Dịch "dropout_vn" - Phần 2

### LINKED ISSUE #944: Dịch "dropout_vn" - Phần 2
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_multilayer-perceptrons/dropout_vn.md#L72-L136

### Hướng dẫn
- [Dịch trực tiếp trên website](https://github.com/aivivn/d2l-vn/blob/master/CONTRIBUTING.md).
- Sau khi nhấn vào phần cần dịch ở trên, hãy chọn **Normal view** và làm theo hướng dẫn ở trên để có thể bắt đầu dịch.

### PR DESCRIPTION
**Pull request by minhduc0711**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Trả lời số issue sau dấu `#` (không có dấu cách): Close #944 

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
    * Các phần in nghiêng, in đậm.
    * Tiêu đề (số lượng dấu `#` đầu dòng).
    * Bảng biểu, chú thích cho bảng (dòng phía trên mỗi bảng bắt đầu bằng dấu `:`).
    * Dịch các từ trong hình vẽ nếu cần. Các dòng có hình có dạng: `caption`.
    * Dịch các chú thích hình vẽ (thay các cụm `*dịch chú thích ảnh phía trên*` bằng bản dịch tiếng Việt).
    * Không dịch các phần code (nằm giữa hai dấu `````).
    * Copy các công thức toán từ bản gốc (các đoạn có `$`).
    * Giữ các dòng gán nhãn (bắt đầu với `:label:`).
    * Không tự thêm bớt các dòng trắng.
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn.

* [x] Tất cả chữ trong các hình vẽ đã được dịch. Nếu chưa, vui lòng xem thêm **[hướng dẫn hoặc yêu cầu dịch hình](https://github.com/aivivn/d2l-vn/blob/master/img/README.md)**.

* [x] Điền tên của bạn vào mục "Những người thực hiện" ở cuối file.

* [ ] **Sau khi được approve**, điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này. Xem danh sách Contributors **[tại đây](https://github.com/aivivn/d2l-vn/blob/master/docs/contributors_info.md)**.

## Với Reviewer

1. Chỉ review nếu Pull Request này đã sẵn sàng cho việc review.

2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

3. **Bạn có thể đề xuất thay đổi cách dịch mỗi dòng trực tiếp như trong hình dưới đây:**
img

### CODE REVIEW

File: chapter_multilayer-perceptrons/dropout_vn.md [hunk 1]
```diff
@@ -92,30 +92,42 @@ to small changed to its inputs.
...
 <!--
 Then, in 2014, Srivastava et al. :cite:`Srivastava.Hinton.Krizhevsky.ea.2014` developed a clever idea for how to apply Bishop's idea to the *internal* layers of the network, too.
 Namely, they proposed to inject noise into each layer of the network before calculating the subsequent layer during training.
 They realized that when training a deep network with many layers, enforcing smoothness just on the input-output mapping.
 -->
 
-*dịch đoạn phía trên*
+Và rồi vào năm 2014, Srivastava et al. :cite:`Srivastava.Hinton.Krizhevsky.ea.2014` đã phát triển một ý tưởng thông minh về cách áp dụng ý tưởng của Bishop cho các tầng *nội bộ* của mạng.
+Cụ thể, họ đề xuất việc thêm nhiễu vào mỗi tầng của mạng trước khi tính toán các tầng kế tiếp trong quá trình huấn luyện.
+Họ nhận ra điều đó khi huấn luyến mạng có nhiều tầng và chỉ ép buộc sự mượt trên phép ánh xạ đầu vào-đầu ra. <!-- WHAT!?? -->
```
Comment (lkhphuc) [inline chapter_multilayer-perceptrons/dropout_vn.md]:
```suggestion
Họ nhận ra rằng nếu chỉ huấn luyến mạng đa tầng với dữ liệu nhiễu, ta chỉ đang ép buộc điều kiện mượt trên phép ánh xạ giữa đầu vào và đầu ra chứ không có ràng buộc mượt giữa các tầng nội bộ.
```
Câu này chắc tác giả viết thiếu ý. Mình hiểu đầy đủ như thế này.

File: chapter_multilayer-perceptrons/dropout_vn.md [hunk 2]
```diff
@@ -124,14 +136,18 @@ The authors argue that neural network overfitting is characterized by a state in
 Dropout, they claim, breaks up co-adaptation just as sexual reproduction is argued to break up co-adapted genes.
 -->
 
-*dịch đoạn phía trên*
+Để nói cho rõ, mối liên kết đến Bishop là của chúng tôi tự đặt ra.
+Đáng ngạc nhiên, bài báo gốc về dropout xây dựng trực giác bằng việc so sánh nó với sinh sản hữu tính.
```
Comment (lkhphuc) [inline chapter_multilayer-perceptrons/dropout_vn.md]:
```suggestion
Đáng ngạc nhiên, bài báo gốc về dropout xây dựng cách hiểu trực giác bằng việc so sánh nó với quá trình sinh sản hữu tính.
```

File: chapter_multilayer-perceptrons/dropout_vn.md [hunk 3]
```diff
@@ -124,14 +136,18 @@ The authors argue that neural network overfitting is characterized by a state in
...
+Họ khẳng định rằng dropout phá bỏ sự đồng thích nghi này, tương tự như việc lập luận rằng sinh sản hữu tính phá bỏ các gen đã đồng thích nghi.
 
 <!--
 The key challenge then is *how* to inject this noise.
 One idea is too inject the noise in an *unbiased* manner so that the expected value of each layer---fixing the others equal to the value it would have taken absent noise.
 -->
 
-*dịch đoạn phía trên*
+Thách thức chính bây giờ là làm thế nào để thêm nhiễu.
+Một ý tưởng là: thêm nhiễu một cách *không thiên lệch* để giá trị kỳ vọng của mỗi tầng <!-- hic câu này chả hiểu gì mọi người giúp với ạ -->
```
Comment (lkhphuc) [inline chapter_multilayer-perceptrons/dropout_vn.md]:
```suggestion
Ta có thể tiêm nhiễu vào theo một cách *không thiên lệch* miễn sao giá trị kì vọng tại mỗi tầng vẫn sẽ bằng giá trị kì vọng của chính tầng đó trước khi được tiêm nhiễu, giả sử rằng các tầng khác đang được giữ nguyên.  
```
Mình hiểu thế này. ông tác giả thiếu 3 dấu - chỗ `so that the expected value of each layer ---fixing the others--- equal to the value it would have taken absent noise.`

File: chapter_multilayer-perceptrons/dropout_vn.md [hunk 4]
```diff
@@ -79,7 +79,7 @@ For now, we turn to the more terrestrial investigation of practical tools that t
 ## Robustness through Perturbations
```
[the remark this answers is not in the document]
Comment (minhduc0711) [reply chapter_multilayer-perceptrons/dropout_vn.md]: yeah từ này mình hiểu nghĩa mà lúc trước không biết dịch thế nào :smile:

File: chapter_multilayer-perceptrons/dropout_vn.md [hunk 5]
```diff
@@ -79,7 +79,7 @@ For now, we turn to the more terrestrial investigation of practical tools that t
...
 -->
 
-## *dịch tiêu đề phía trên*
+## Sự mạnh mẽ thông qua Nhiễu loạn
```
Comment (minhduc0711) [inline chapter_multilayer-perceptrons/dropout_vn.md]:
```suggestion
## Sự kiên định thông qua Nhiễu loạn
```

File: chapter_multilayer-perceptrons/dropout_vn.md (hunk 1 above)
Comment (minhduc0711) [reply chapter_multilayer-perceptrons/dropout_vn.md]: woah sao đoán ý hay vậy :))) mình sửa lại một chút thôi
```suggestion
Họ nhận ra rằng khi huấn luyến mạng đa tầng với dữ liệu nhiễu, ta chỉ đang ép buộc điều kiện mượt trên phép ánh xạ giữa đầu vào và đầu ra chứ không có ràng buộc mượt giữa các tầng nội bộ.
```

File: chapter_multilayer-perceptrons/dropout_vn.md (hunk 3 above)
Comment (minhduc0711) [reply chapter_multilayer-perceptrons/dropout_vn.md]: à hiểu rồi, hôm đấy do mình không đọc kĩ đoạn dưới :)))
mình nghĩ vẫn nên dịch là `thêm nhiễu` để nghe đỡ lạ
```suggestion
Ta có thể thêm nhiễu một cách *không thiên lệch* sao cho giá trị kỳ vọng của mỗi tầng bằng giá trị kỳ vọng của chính tầng đó trước khi được thêm nhiễu, giả sử rằng các tầng khác được giữ nguyên.  
```

File: chapter_multilayer-perceptrons/dropout_vn.md [hunk 6]
```diff
@@ -474,7 +490,7 @@ với dấu `@` ở đầu. Ví dụ: @aivivn.
 *
 
 <!-- Phần 2 -->
-*
+* Phạm Minh Đức
```
Comment (minhduc0711) [inline chapter_multilayer-perceptrons/dropout_vn.md]:
```suggestion
* Phạm Minh Đức
* Lê Khắc Hồng Phúc
```

File: chapter_multilayer-perceptrons/dropout_vn.md [hunk 7]
```diff
@@ -79,7 +79,7 @@ For now, we turn to the more terrestrial investigation of practical tools that t
 ## Robustness through Perturbations
 -->
 
-## *dịch tiêu đề phía trên*
+## Sự kiên định thông qua Nhiễu loạn
```
Comment (tiepvupsu) [inline chapter_multilayer-perceptrons/dropout_vn.md]: Thảo luận thêm tại https://d2l-vn.slack.com/archives/CR68J40BH/p1584753958111000

File: chapter_multilayer-perceptrons/dropout_vn.md [hunk 8]
```diff
@@ -92,30 +92,42 @@ to small changed to its inputs.
 For instance, when we classify images, we would expect that adding some random noise to the pixels should be mostly harmless.
 -->
 
-*dịch đoạn phía trên*
+Hãy cùng nghĩ một chút về thứ mà ta mong đợi từ một mô hình dự đoán tốt.
+Ta muốn mô hình hoạt động tốt khi gặp dữ liệu mà nó chưa từng thấy.
+Lý thuyết khái quát cổ điển cho rằng: để thu hẹp khoảng cách giữa chất lượng khi huấn luyện và chất lượng khi kiểm tra, ta nên hướng tới một mô hình *đơn giản*.
+Sự đơn giản này có thể nằm ở việc số chiều thấp, điều mà chúng ta đã khám phá khi thảo luận về hàm cơ sở đơn thức trong mô hình tuyến tính ở :numref:`sec_model_selection`. 
+Như ta đã thấy khi bàn về phân rã trọng số (điều chuẩn $\ell_2$) ở :numref:`sec_weight_decay`, chuẩn (nghịch) của các tham số là một phép đo khác cho sự đơn giản.
```
Comment (tiepvupsu) [inline chapter_multilayer-perceptrons/dropout_vn.md]:
```suggestion
Như ta đã thấy khi bàn về suy giảm trọng số (điều chuẩn $\ell_2$) ở :numref:`sec_weight_decay`, (nghịch đảo) chuẩn của các tham số là một phép đo khác cho sự đơn giản.
```

Glossary `weight decay` là `suy giảm trọng số`.

File: chapter_multilayer-perceptrons/dropout_vn.md (hunk 7 above)
Comment (tiepvupsu) [reply chapter_multilayer-perceptrons/dropout_vn.md]: Mình thấy dịch `Khả năng Kháng Nhiễu` khá ok.

Comment (minhduc0711) [reply chapter_multilayer-perceptrons/dropout_vn.md]: ooops quên mất cái PR này, em vừa update rồi ạ

File: chapter_multilayer-perceptrons/dropout_vn.md [hunk 9]
```diff
@@ -124,14 +136,18 @@ The authors argue that neural network overfitting is characterized by a state in
 Dropout, they claim, breaks up co-adaptation just as sexual reproduction is argued to break up co-adapted genes.
 -->
 
-*dịch đoạn phía trên*
+Để nói cho rõ, mối liên kết đến Bishop là của chúng tôi tự đặt ra.
+Đáng ngạc nhiên, bài báo gốc về dropout xây dựng cách hiểu trực giác bằng việc so sánh nó với quá trình sinh sản hữu tính.
+Các tác giả cho rằng hiện tượng quá khớp mạng nơ-ron là biểu hiện của việc mỗi tầng đều dựa vào một khuôn mẫu nhất định của các giá trị kích hoạt ở tầng trước đó, và họ gọi trạng thái này là `đồng thích nghi`.
```
Comment (duythanhvn) [inline chapter_multilayer-perceptrons/dropout_vn.md]:
```suggestion
Các tác giả cho rằng hiện tượng quá khớp mạng nơ-ron là biểu hiện của việc mỗi tầng đều dựa vào một khuôn mẫu nhất định của các giá trị kích hoạt ở tầng trước đó, và họ gọi trạng thái này là *đồng thích nghi*.
```
Lại sáng tạo rồi...

Comment (minhduc0711) [reply chapter_multilayer-perceptrons/dropout_vn.md]: whoops sorry ạ :rofl:
